### DISCUSSION TITLE
Error? in startup, diffuses image though..

### DISCUSSION THREAD

**Discussion by electricatom369** [comfyanonymous/ComfyUI#353] [Q&A]
start up error?:
 ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\python_embeded\lib\site-packages\torch\storage.py:899: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly. To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage() storage = cls(wrap_storage=untyped_storage)

**Comment by comfyanonymous** [comment]
It's a torch warning you can ignore that.

**Comment by electricatom369** [comment]
Thank you Sir. :)
